pkg/errors/factory.go

```diff
@@ -27,29 +27,39 @@ func isCRDError(status api_errors.APIStatus) bool {
 			return true
 		}
 	}
-
 	return false
 }
 
+func isNoRouteToHostError(err error) bool {
+	return strings.Contains(err.Error(), "no route to host") || strings.Contains(err.Error(), "i/o timeout")
```

Comment:
basing actions on error descriptions is quite fragile. I wonder whether there is a better, typed approach ?

We can also think about it later.